src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/convert_to_plugin_specific_node.cpp

```diff
@@ -14,6 +14,7 @@ namespace test {
 
 using ConvertToPluginSpecificNodeParams = std::tuple<ov::Shape,                      // non const input shape
                                                      ov::Shape,                      // const input shape
+                                                     bool,                           // large const number
```

Comment:
Do we really need separate param? Can we extend the range for random generator by default?

Reply:
Good point! Applied. Thanks Dmitry!